app/scripts/components/exploration/atoms/selectedForEditing.ts

```diff
@@ -0,0 +1,12 @@
+import { atomWithUrlValueStability } from '$utils/params-location-atom/atom-with-url-value-stability';
```

Comment:
Doesn't it make more sense to keep this atom with other aoi-related atoms (in `common/map/controls/aoi/atoms.ts`)?

Reply:
Makes sense, moved 👍